sys/include/timex.h

```diff
@@ -53,6 +53,11 @@ extern "C" {
  */
 #define US_PER_MS  (1000U)
 
+/**
+ * @brief The number of microseconds per centisecond
+ */
+#define US_PER_CS  (10000U)
```

Comment:
I this really needed? centisecond is not widely used in general and you are the first in RIOT. I would rather move this to the header/source file where you use it for now.

Reply:
DHCPv6 uses hundredth of a second (aka centiseconds) and there used to be discussions about introducing centiseconds to timex.h for routing protocols which also use hundredth of a second sometimes (see https://github.com/RIOT-OS/RIOT/pull/5158). I could move it to private, but I believe there is a benefit to it.

Also, yes, the term centiseconds is quite quaint, but at least "hundredth of a second" is used quite often (e.g. in sports)